orcid-web/src/main/java/org/orcid/frontend/web/controllers/PublicProfileController.java

```diff
@@ -505,6 +505,19 @@ private void validateVisibility(Visibility v) {
                 List<AffiliationGroupForm> elementsFormList = new ArrayList<AffiliationGroupForm>();
                 IntStream.range(0, elementsList.size()).forEach(idx -> {
                     AffiliationGroupForm groupForm = AffiliationGroupForm.valueOf(elementsList.get(idx), type.name() + '_' + idx, orcid);
+                    // Fill country on the default affiliation
+                    AffiliationForm defaultAffiliation = groupForm.getDefaultAffiliation();
+                    if (defaultAffiliation != null) {
+                        // Set country name
+                        defaultAffiliation.setCountryForDisplay(groupForm.getDefaultAffiliation().getCountry().getValue());
```

Comment:
Hey @lizkrznarich, you should check if  `groupForm.getDefaultAffiliation().getCountry() != null` right?

Reply:
@amontenegro Was also missing from AffiliationsController - can you take a look at the changes I just pushed on https://github.com/ORCID/ORCID-Source/pull/5518/commits/1a2565cc7e7cf0d21428d270673f4bb09e5c6d4c ?